Library/Homebrew/cask/audit.rb

```diff
@@ -563,14 +563,19 @@ def check_livecheck_min_os
 
       return if xml.blank?
 
-      item = xml.get_elements("//rss//channel//item").first
+      item = xml.elements["//rss//channel//item"]
       return if item.blank?
 
-      min_os = item.elements["sparkle:minimumSystemVersion"].text
+      min_os = item.elements["sparkle:minimumSystemVersion"]&.text
       return if min_os.blank?
 
-      min_os_string = OS::Mac::Version.new(min_os).strip_patch
-      cask_min_os = cask.depends_on.macos.version
+      begin
+        min_os_string = OS::Mac::Version.new(min_os).strip_patch
+      rescue MacOSVersionError
+        return
+      end
```

Comment:
Since we appear to be flagging casks that don't have any `depends_on macos:` (which I assume is intended - I can see it being useful), it might be worth doing:

```suggestion
      end
      
      return if min_os_string == MacOS::Version::OLDEST_ALLOWED
```

Since `depends_on macos: ">= :el_capitan"` does nothing given `brew` can't run on < El Capitan.